scripts/build

```diff
@@ -19,9 +19,9 @@ LINKFLAGS="-X github.com/k3s-io/kine/pkg/version.Version=$VERSION"
 LINKFLAGS="-X github.com/k3s-io/kine/pkg/version.GitCommit=$COMMIT $LINKFLAGS"
 
 echo Building Kine
-CGO_CFLAGS="-DSQLITE_ENABLE_DBSTAT_VTAB=1 -DSQLITE_USE_ALLOCA=1" go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/kine
+CGO_CFLAGS="-DSQLITE_ENABLE_DBSTAT_VTAB=1 -DSQLITE_USE_ALLOCA=1" go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -tags nats -o bin/kine
 if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
-    GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/kine-darwin
-    GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/kine-windows
-    GOOS=linux go build -ldflags "$LINKFLAGS" -o bin/kine-amd64
+    GOOS=darwin go build -ldflags "$LINKFLAGS" -tags nats -o bin/kine-darwin
+    GOOS=windows go build -ldflags "$LINKFLAGS" -tags nats -o bin/kine-windows
+    GOOS=linux go build -ldflags "$LINKFLAGS" -tags nats -o bin/kine-amd64
```

Comment:
I presume this is only building the binaries for CI, correct? Otherwise if we want to have an opt-in embedded nats build using this tag, what is the precedent?

Reply:
This builds the artifacts that are used both for CI, and for release to GH and the Docker images. I am fine with having it on by default here; K3s embeds kine as a library and doesn't use the artifacts.